### PR TITLE
Colby Pearce

### PR DESCRIPTION
**Pull request by Crpearce**
Here is my finished M1 Prework!

### CODE REVIEW

File: section2/reflection.md [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do. Use either && or || i.e. logical AND or logical OR operator to connect two or more conditions inside a if statement. you just need to make sure your conditions are wrapped in parenthesis
 
-6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
+6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code? if else statements
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+7. What questions do you still have about `if` statements and/or functions? i still feel a little bit fuzzy making sure that I am writing out my coding correct for if else statements.  I think this should improve with more practice and getting a better grasp of the logic behind it
```
Comment (SageOfCode) [inline section2/reflection.md:26]: Don't worry too much if this doesn't feel familiar yet, you will get a lot of practice with if statements in the program and after. Your if statements look great.

File: section3/exercises/arrays.js [hunk 2]
```diff
@@ -106,11 +135,18 @@ var hasCharger = true;
 var chargerType = "car";
 
 // Write your conditional here
-
+//This line below is stating they can call an uber if they have greater than or equal to 15% battery
+if (percentBatteryLeft >= 15) {
+  console.log('Can call uber');
+}
+// This next line will determine you can also call an uber if you both have a charger and it is a car charger
+else if (hasCharger && chargerType) {
```
Comment (SageOfCode) [inline section3/exercises/arrays.js:143]: Putting this line after the closing bracket on line 141 is the more standard syntax.

File: section3/reflection.md [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
 
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
+The purpose of a loop is to help perform repeated tasks that have been entered without having to write the code over and over again. Using a loop in conjunction with an array can allow for you to run a loop for specific items from an extended list.
 
 5. If you had to teach a 5 year old about loops, how would you explain it?
+I would explain a loop to a 5 year old similarly to a scavenger hunt.  You have a list of all the potential objectives or tasks outlined.  You complete the tasks by following the instructions over and over by process of elimination until you are complete.
 
 6. What questions do you still have about Arrays and loops?
+My main questions about arrays and loops is just the general writing of them.  I know this is still very basic stuff in regards to writing arrays and loops but I am still struggling to comprehend how to write out variables.  I am also still having questions in regards to completing my for statement correctly. I was wasting a lot of time tinkering around trying to get the loops to perform like I was hoping.
```
Comment (SageOfCode) [inline section3/reflection.md:18]: Your syntax looks good overall, I wold suggest just getting more practice between now and your cohort start date.

**Review by SageOfCode — COMMENTED** [review]
Great work here @Crpearce! Please get the missing link for the note taking reflection gist added at your earliest convenience. It looks like you have a great grasp over the material introduced in mod 0.

**Comment by Crpearce** [comment]
Good Morning Sage,

Sorry about the missing link, I am not sure what happened there it looked
like it was going to the original gist. I have updated my repository now
with the link I had worked on. Do I need to create a new pull request for
this?

**Comment by SageOfCode** [comment]
@Crpearce Thank you for getting that updated. You are good to go, no need for a new pull request.

**Pull request closed without merging**